doc/ref_impl/cntt-ri/chapters/chapter07.md

```diff
@@ -30,9 +30,22 @@ It is assumed that the reader of this chapter has the skill set to install Commo
 <a name="7.2"></a>
 ## 7.2 Prerequisites
 
-The following are pre-requisites to be completed in advance of software deployments:
+The following hardware was cabled and set up according to the OPNFV Pharos Specification:
+
+| Node  | CPU Model | Memory | HDD           | SSD         | 1 GbE NIC | 10 GbE NIC |
+|-------|-----------|--------|---------------|-------------|-----------|------------|
+| Jump  | 2xE5-2699 | 64 GB  | 1 x 3 TB Sata | 1 x 180 SSD | 2         | 2          |
+| 1     | 2xE5-2699 | 64 GB  | 1 x 3 TB Sata | 1 x 180 SSD | 2         | 2          |
+| 2     | 2xE5-2699 | 64 GB  | 1 x 3 TB Sata | 1 x 180 SSD | 2         | 2          |
+| 3     | 2xE5-2699 | 64 GB  | 1 x 3 TB Sata | 1 x 180 SSD | 2         | 2          |
+| 4     | 2xE5-2699 | 64 GB  | 1 x 3 TB Sata | 1 x 180 SSD | 2         | 2          |
+| 5     | 2xE5-2699 | 64 GB  | 1 x 3 TB Sata | 1 x 180 SSD | 2         | 2          |
+
```

Comment:
@markbeierl 
Can we identify which version of E5-2699 ?, is it v4 or v3 ? as number of cores is different